toolchain/lower/constant.cpp

```diff
@@ -239,6 +209,24 @@ static auto EmitAsConstant(ConstantContext& /*context*/,
   CARBON_FATAL("TODO: Add support: {0}", inst);
 }
 
+template <typename InstT>
```

Comment:
Maybe add a comment explaining this...

Though I'll note this strays a little from the original change (as does TypeStore). Some of this feels like it would've been nicer split out, but I'm basically done reviewing now. :-P

Reply:
Yeah, sorry about that. (I started with the `TypeStore` refactoring and ended up cleaning up too much other stuff "in passing", ending up with this mostly being a change to base / adapt declarations instead.) :(

Comment added.